graphene_django/static/graphene_django/graphiql.js

```diff
@@ -97,6 +97,6 @@
   // Render <GraphiQL /> into the body.
   ReactDOM.render(
     React.createElement(GraphiQL, options),
-    document.body
+    document.getElementsByClassName("editor")[0]
```

Comment:
Minor point but I think it's better to select by ID instead of class since there is only going to be 1 editor on a page.

```suggestion
    document.getElementById("editor")
```